bigtable/client/readrowsparser_test.cc

```diff
@@ -0,0 +1,2054 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "bigtable/client/readrowsparser.h"
+
+#include <google/protobuf/text_format.h>
+#include <gtest/gtest.h>
+
+#include <numeric>
+#include <sstream>
+#include <vector>
+
+using google::bigtable::v2::ReadRowsResponse_CellChunk;
+
+namespace bigtable {
+namespace {
+
+std::string LabelsToString(const std::vector<std::string>& labels) {
```

Comment:
Since we now have `absl::StrJoin()` do you want to use it here?

Reply:
yay!